sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlobClientBase.java

```diff
@@ -789,10 +1090,35 @@ public Response<String> copyFromUrlWithResponse(String copySource, Map<String, S
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Response<String> copyFromUrlWithResponse(BlobCopyFromUrlOptions options, Duration timeout,
         Context context) {
```

Comment:
Do we need to handle any `null` context objects here? I noticed this was handled in the `abortCopyFromUrlWithResponse` method but noticed it was not set to `Context.NONE` here if it is `null`.

Reply:
Thanks for catching this! Will add the check for whether context is null.